### PR TITLE
Bump up some of our deps to latest and fix a jade error.

### PR DESCRIPTION
**Pull request by andyburke**
Bumps packages.
Fixes a jade error regarding duplicate alt attributes.
Fixes a warning from jade about missing spaces.

### CODE REVIEW

**Comment by d48** [comment]
lgtm

**Comment by jcblw** [comment]
:+1: I can get this merged tonight after I test it out.

File: public/components/speakers/_speaker.jade [hunk 1]
```diff
@@ -14,8 +14,7 @@ li.jsla-speaker.clearfix( id=github )
                         i.icon-github
         div.jsla-speaker-content.layer-2
             h2.jsla-speaker-title= title
-            p.jsla-speaker-description
-                !{description}
+            p.jsla-speaker-description !{description}
```
Comment (TimothyGu) [inline public/components/speakers/_speaker.jade]: How about just:

```jade
            p.jsla-speaker-description!= description
```

?

Comment (andyburke) [inline public/components/speakers/_speaker.jade]: Sure? I don't know jade. :stuck_out_tongue_winking_eye:

**Comment by andyburke** [comment]
So does this matter anymore or does #166 deprecate this?

**Comment by jcblw** [comment]
hmmm didnt see that the was an extra commit on here. will merge

**Pull request merged**